yt_dlp/extractor/zaiko.py

```diff
@@ -45,7 +46,13 @@ class ZaikoIE(ZaikoBaseIE):
             'uploader_id': '454',
             'uploader': 'ZAIKO ZERO',
             'release_timestamp': 1583809200,
-            'thumbnail': r're:https://[a-z0-9]+.cloudfront.net/[a-z0-9_]+/[a-z0-9_]+',
+            'thumbnails': [{
+                'id': r're:[a-z0-9_]+',
+                'url': r're:https://[a-z0-9]+.cloudfront.net/[a-z0-9_]+/[a-z0-9_]+',
+            }, {
+                'id': r're:[a-z0-9_]+',
+                'url': r're:https://media.zaiko.io/[a-z0-9_]+/[a-z0-9_]+',
+            }],
```

Comment:
I know you can't actually run this test currently, but if you could, the test would still check for `thumbnail` and ignore `thumbnails`, so let's just do this
```suggestion
            'thumbnail': r're:^https://[\w.-]+/\w+/\w+',
            'thumbnails': 'count:2',
```
and add a `skip` key to this test with the reason as the value